topics/ecology/tutorials/gbif_cleaning/tutorial.md

```diff
@@ -0,0 +1,367 @@
+- How can I get ecological data from GBIF?
+- How do I check and clean the data from GBIF?
+
+- Which ecoinformatics techniques are important to know for this type of data?
+  - Data handling
+  - Data filtering
+  - Data visualization, notably dealing with GIS data
+objectives:
+- Get occurrence data on a species
+- Visualize the data to understand them
+- Clean GBIF dataset for further analyses
+time_estimation: 0H30M
+key_points:
+- Take the time to look at your data first, manipulate it before analyzing it
+contributors:
+- yvanlebras
+- sbenateau
+
+---
+
+
+# Introduction
+
+
+GBIF (Global Biodiversity Information Facility, www.gbif.org) is for sure THE most remarkable biodiversity data aggregator worldwide giving access to more than 1 billion reords across all taxonomic groups. The data provided via these sources are highly valuable for research. However, some issues exist concerning data heterogeneity, as they are obtained from various collection methods and sources.
+
+In this tutorial we will propose a way to clean occurence records retrieved from GBIF.
+
+This tutorial is based on the Ropensci {% cite zizka2018 %} tutorial.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. Download data from GBIF
+> 2. Clean data
+> 3. Convert text data into GIS format
+> 4. Visualize spatialized data
+> {:toc}
+>
+{: .agenda}
+
+# Retrive data from GBIF
+
+## Get data
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from GBIF
+>
+> 3. **Get species occurrences data** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Scientific name of the species"*: write the scientific name of something you are interested on, for example `Loligo vulgaris`
+>    - *"Data source to get data from"*: `Global Biodiversity Information Facility : GBIF`
+>    - *"Number of records to return"*: `999999` is a minimum value
+>
+>    > <comment-title></comment-title>
+>    >
+>    > The spocc Galaxy tool allows you to search species occurences across a single or many data sources (GBIF, eBird, iNaturalist, EcoEngine, VertNet, BISON). Changing the number of records to return allows you to have all or limited numbers of occurences. Specifying more than one data source will change the manner the output dataset is formatted.
+>    >
+>    {: .comment}
+>
+>
+> 4. Check the datatype
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add to the output dataset (and as a good practice, each dataset you generate on Galaxy) a tag and/or a propagated tag (beginning by a `#`, here corresponding to the species (#LoligoVulgaris for example here) and/or the data source (#GBIF for example here). This will help you finding content of particular interest (using the filtering option on the history search form for example) + visualize (notably if you are using propagated tag) which dataset is associated to which content.
```

Comment:
very long sentence. Maybe you can write short instructions, then do a sentence of why this is useful below the instructions?

Reply:
a first try! Thank you!